src/System.Net.HttpListener/tests/AuthenticationTests.cs

```diff
@@ -0,0 +1,154 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Xunit;
+
+namespace System.Net.Tests
+{
+    // TODO: #13187
```

Comment:
Contributing to the source code shouldn't require permanent Internet access (I know it does and I don't like it :) ).
Could be something like `// TODO: #13187 - Missing non-Windows implementation.`